--- tesseract-2.01/dict/dawg.h	2007-07-06 20:48:31 +0200
+++ tesseract-2.01/dict/dawg.h.new	2010-04-24 14:20:02 +0200
@@ -68,17 +68,10 @@
 #define FLAG_START_BIT         8
 #define NEXT_EDGE_START_BIT    11
 
-#ifdef __MSW32__
-#define NO_EDGE                (INT64) 0x001fffffffffffffi64
-#define NEXT_EDGE_MASK         (INT64) 0xfffffffffffff800i64
-#define FLAGS_MASK             (INT64) 0x0000000000000700i64
-#define LETTER_MASK            (INT64) 0x00000000000000ffi64
-#else
 #define NO_EDGE                (INT64) 0x001fffffffffffffll
 #define NEXT_EDGE_MASK         (INT64) 0xfffffffffffff800ll
 #define FLAGS_MASK             (INT64) 0x0000000000000700ll
 #define LETTER_MASK            (INT64) 0x00000000000000ffll
-#endif
 
 #define MAX_NUM_EDGES_IN_SQUISHED_DAWG_FILE 2000000
 
