diff -urw faad2-2.7.orig/Makefile.am faad2-2.7/Makefile.am
--- faad2-2.7.orig/Makefile.am	Tue Jul 27 05:52:20 2004
+++ faad2-2-7/Makefile.am	Wed Mar  5 14:28:20 2008
@@ -1,4 +1,4 @@
-SUBDIRS = libfaad common frontend plugins
+SUBDIRS = libfaad
 
 EXTRA_DIST = faad2.spec
 
diff -urw faad2-2.7.orig/configure.in faad2-2.7/configure.in
--- faad2-2.7.orig/configure.in	Mon Sep 25 16:52:53 2006
+++ faad2-2.7/configure.in	Wed Mar  5 14:28:25 2008
@@ -154,14 +154,6 @@
 fi
 
 AC_CONFIG_FILES(libfaad/Makefile)
-AC_CONFIG_FILES(common/Makefile)
-AC_CONFIG_FILES(common/mp4ff/Makefile)
-AC_CONFIG_FILES(plugins/Makefile)
-AC_CONFIG_FILES(plugins/xmms/Makefile)
-AC_CONFIG_FILES(plugins/xmms/src/Makefile)
-AC_CONFIG_FILES(plugins/mpeg4ip/Makefile)
-AC_CONFIG_FILES(faad2.spec)
-AC_CONFIG_FILES(frontend/Makefile)
 AC_CONFIG_FILES(Makefile)
 
 AC_OUTPUT
